### ISSUE TITLE
[$15][Safari] [voiceover] Contact us field title are read twice

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3081]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing 
2. click  support button 
3. hit tab key 
4. Observe

**Expected Result**
Title should only read once by voiceover

**Actual Result**
Field title are read twice by voiceover

**Device**: Desktop/Labtop
**Browser**: Safari

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Voiceover
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=09f22811%2D138e%2D4d3b%2Da8d5%2Dd3693a709128&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2FCreeya%2Fbug%2D14

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3081 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3081 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup. PR against `qa-accessibility` branch.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097878 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
